package/MDAnalysis/lib/formats/cython_util.pyx

```diff
@@ -110,7 +110,11 @@ cdef cnp.ndarray ptr_to_ndarray(void* data_ptr, cnp.int64_t[:] dim, int data_typ
     array_wrapper = ArrayWrapper()
     array_wrapper.set_data(<void*> data_ptr, <int*> &dim[0], dim.size, data_type)
 
-    cdef cnp.ndarray ndarray = np.array(array_wrapper, copy=False)
+    if np.__version__[0] == "2":
+        copy = None
+    else:
+        copy = False
```

Comment:
A more concise/idiomatic upstream approach was applied in the revised version of my patch in case we care about that: https://github.com/scipy/scipy/pull/20172/files#diff-d28fbb0be287769c763ce61b0695feb0a6ca0e67b28bafee20526b46be7aaa84R59

Basically, abstracting the check to avoid duplication and using `np.lib.NumpyVersion` instead of pep440 or a raw string check like here. We probably don't need to worry about `2.0.0dev0` versions though, so we may care less about that, though the version check duplication I've added more broadly is indeed a bit much.